tests/Fixtures/resources.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<resources>
+    <configdummy>
```

Comment:
Actually, we could change most of them into attributes. It'd be more correct.

Also, we should use

```xml
<operation name="my_op_name">
```